### PR TITLE
Feat/study page add search

### PR DESCRIPTION
**Pull request by adipaul1981**
image

### CODE REVIEW

**Review by celinepelletier — APPROVED** [review]

File: src/pages/studies/SidebarFilters.tsx [hunk 1]
```diff
@@ -47,6 +70,23 @@ const SidebarFilters = ({ studiesResults, studiesMappingResults, onChange }: Own
 
   return (
     <>
```
Comment (evans-g-crsj) [inline src/pages/studies/SidebarFilters.tsx:72]: We must use colours from the theme via scss

File: src/pages/studies/SidebarFilters.tsx [hunk 2]
```diff
@@ -47,6 +70,23 @@ const SidebarFilters = ({ studiesResults, studiesMappingResults, onChange }: Own
...
+      <div style={{ color: colors.iconColor }}>Search</div>
+      <Input
+        prefix={<SearchOutlined style={{ color: colors.iconColorDisabled }} />}
+        placeholder="Search..."
+        onPressEnter={(e: any) => {
```
Comment (evans-g-crsj) [inline src/pages/studies/SidebarFilters.tsx:77]: if possible, try to get rid of `any` and replace it with correct type.

File: src/pages/studies/utils.ts [hunk 3]
```diff
@@ -81,10 +85,12 @@ export const updateQueryFilters = (
   history: any,
   field: string,
   filters: TSqonGroupContent,
+  operator: TSqonGroupOp = 'and',
 ): void => {
   const currentFilter = getFiltersQuery();
 
-  let newFilters: ISqonGroupFilter | Record<string, never> = { content: filters, op: 'and' };
+  let newFilters: ISqonGroupFilter | Record<string, never> = { content: filters, op: operator };
```
Comment (evans-g-crsj) [inline src/pages/studies/utils.ts:92]: isn't it a Sqon or SqonFilter? I do not understand the Record<string, never>

File: src/pages/studies/SidebarFilters.tsx [hunk 4]
```diff
@@ -32,6 +35,26 @@ const keyEnhanceBooleanOnly = (key: string) => {
   }
 };
 
+const onSearch = (search: string) => {
+  const searchFields = ['search'];
+  searchFields.forEach((f) =>
```
Comment (evans-g-crsj) [inline src/pages/studies/SidebarFilters.tsx:40]: why using a `foreach` since there's only one element?

**Review by evans-g-crsj — CHANGES_REQUESTED** [review]

File: src/pages/studies/SidebarFilters.tsx (hunk 4 above)
Comment (adipaul1981) [reply src/pages/studies/SidebarFilters.tsx:40]: good call . yeah there were a couple b4 and they were merged into a single new field (search). Ill refactor.

**Pull request merged**